airflow/providers/databricks/operators/databricks.py

```diff
@@ -88,6 +88,19 @@ def _handle_databricks_operator_execution(operator, hook, log, context) -> None:
                             f"{operator.task_id} failed with terminal state: {run_state} "
                             f"and with the error {run_state.state_message}"
                         )
+                    if isinstance(operator, DatabricksRunNowOperator) and operator.repair_run:
+                        operator.repair_run = False
+                        log.warning(
+                            error_message + "but since repair run is set, repairing the run with all "
+                            "failed tasks"
```

Comment:
```suggestion
                            "%s but since repair run is set, repairing the run with all failed tasks",
                            error_message
```

Better to avoid concatenation in logging message and use %-substitution